plugins/tech-insights-backend/README.md

```diff
@@ -91,7 +91,15 @@ const myFactRetrieverRegistration = createFactRetrieverRegistration(
 );
 ```
 
-Then you can modify the example `techInsights.ts` file shown above like this:
```

Comment:
ping @Xantier - while bumping to the latest version of Backstage, I noticed the breaking change on the API of `createFactRetrieverRegistration` which wasn't updated in the README

Reply:
Thanks, I added changes to that in https://github.com/backstage/backstage/pull/9094